project/AppDependencies.scala

```diff
@@ -2,28 +2,29 @@ import sbt.*
 
 object AppDependencies {
 
-  private val hmrcMongoPlayVersion = "1.6.0"
-  private val bootstrapVersion     = "7.23.0"
+  private val hmrcMongoPlayVersion = "1.8.0"
+  private val bootstrapVersion     = "8.5.0"
 
   val compile: Seq[ModuleID] = Seq(
-    "uk.gov.hmrc"                  %% "bootstrap-frontend-play-28" % bootstrapVersion,
-    "uk.gov.hmrc.mongo"            %% "hmrc-mongo-play-28"         % hmrcMongoPlayVersion,
-    "uk.gov.hmrc"                  %% "play-frontend-hmrc-play-28" % "8.5.0",
+    "uk.gov.hmrc"                  %% "bootstrap-frontend-play-30" % bootstrapVersion,
+    "uk.gov.hmrc.mongo"            %% "hmrc-mongo-play-30"         % hmrcMongoPlayVersion,
+    "uk.gov.hmrc"                  %% "play-frontend-hmrc-play-30" % "8.5.0",
     "org.typelevel"                %% "cats-core"                  % "2.10.0",
     "com.digitaltangible"          %% "play-guard"                 % "2.5.0",
```

Comment:
This isn't the play 3 version of the library

https://github.com/sief/play-guard?tab=readme-ov-file#getting-play-guard